javascript/javascript3/week2/.gitignore

```diff
@@ -0,0 +1 @@
+.history/*
```

Comment:
It would be better if you put this file in the root of your `hyf-homework` folder instead! Then it will apply to all branches and commits you make.

Reply:
ok will do that! which branch should I use?